core/kernel/dt.c

```diff
@@ -125,6 +125,39 @@ int dt_map_dev(const void *fdt, int offs, vaddr_t *base, size_t *size)
 	return 0;
 }
 
+int dt_map_secure_dev(const void *fdt, int offs, vaddr_t *base, size_t *size)
+{
+	paddr_t pbase;
+	vaddr_t vbase;
+	size_t sz;
+	int st;
```

Comment:
`= 0;`

Reply:
Should we go to modify the `dt_map_dev()` too as this was more or less copy from there?

Reply:
Looking at the file more. It is full of missing variable initializations. Thou there does not seem to be problematic cases.

Should I make fix it all commit before adding a new one? Or shall we continue with existing style within the file?